daceml/onnx/schema.py

```diff
@@ -93,7 +93,9 @@ def from_json(cls, json, parent=None):
             return cls(**constructor_args)
 
         def to_json(self):
-            return dace.serialize.all_properties_to_json(self)
+            serialized = dace.serialize.all_properties_to_json(self)
+            serialized["type"] = cls.__name__
```

Comment:
still necessary?

Reply:
We don't use it, but it does improve the situation